map_view/display-device-location/README.metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+    "category": "Map",
+    "description": "Display your current position on the map, as well as switch between different types of auto pan modes.",
+    "ignore": false,
+    "images": [
+        "DisplayDeviceLocation.png"
+    ],
+    "keywords": [
+        "GPS",
+        "compass",
+        "location",
+        "map",
+        "mobile",
+        "navigation",
+        "ArcGISMap",
+        "LocationDisplay",
+        "LocationDisplay.AutoPanMode",
+        "MapView",
+        "SimulatedLocationDataSource",
+        "SimulationParameters"
+    ],
+    "redirect_from": [
+        "/java/latest/sample-code/display-a-map.htm"
+    ],
+    "relevant_apis": [
+        "ArcGISMap",
+        "LocationDisplay",
+        "LocationDisplay.AutoPanMode",
```

Comment:
@Rachael-E might be able to tell whether enums/method calls are safe to put in this section, as it is used for generating some linkes on the user webpages?